addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3662,12 +3662,9 @@ var SnippetsMenu = Widget.extend({
         // snippet will leave the tooltip "forever" visible even if the "Hide in
         // Desktop" button is gone.
         const tooltipClass = 'aria-describedby';
-        const tooltippedEl = this.el.querySelector(`[${tooltipClass}^="tooltip"]`);
-        if (tooltippedEl) {
-            const tooltipEl = document.getElementById(tooltippedEl.getAttribute(tooltipClass));
-            if (tooltipEl) {
-                Tooltip.getInstance(tooltipEl).hide();
-            }
+        const tooltipEl = this.el.querySelector(`[${tooltipClass}^="tooltip"]`);
+        if (tooltipEl) {
+            Tooltip.getInstance(tooltipEl).hide();
```

Comment:
Hello @Guillaume-gdi 👋 

I would like to have your opinion on this PR because it changes some code that you wrote and that seemed to be very deliberately written as it was in: https://github.com/odoo/odoo/pull/85666/files#diff-52a4f9d2c217548e69e6b7fd097f286f1754a6389734eea254b87255e501cbefR3458